src/app/pages/FrontPage/index.test.jsx

```diff
@@ -131,19 +110,35 @@ jest.mock('#containers/PageHandlers/withContexts', () => Component => {
   );
 });
 
+const togglesUrl =
+  'https://mock-toggles-endpoint.bbc.co.uk/?application=simorgh&service=mundo&__amp_source_origin=https://www.test.bbc.com';
+
 describe('Front Page', () => {
   describe('snapshots', () => {
     it('should render a pidgin frontpage correctly', async () => {
+      fetch.mockResponse(JSON.stringify(pidginFrontPageData));
```

Comment:
I think for consistency, we should use fetch-mock throughout this file. So I think we can replace this line with something like:

```
      fetchMock.mock(
        'http://localhost/some-front-page-path.json',
        JSON.stringify(pidginFrontPageData),
      );

```

like what you've done below.

Reply:
Totally 👍